extensions/amp-analytics/0.1/vendors.js

```diff
@@ -107,6 +107,44 @@ export const ANALYTICS_CONFIG = {
       'xhrpost': false,
       'image': true
     }
+  },
+
+  'chartbeat': {
+    'vars': {
+      'decayTime': 15
+    },
+    'requests': {
+      'host': 'https://ping.chartbeat.net',
+      'interval': '${host}/ping?h=${domain}&' +
+        'p=${canonicalPath}&' +
+        'u=${clientId}&' +
+        'd=${canonicalHost}&' +
+        'g=${uid}&' +
+        'g0=${sections}&' +
+        'g1=${authors}&' +
+        'g2=${zone}&' +
+        'g3=${sponsorName}&' +
+        'g4=${contentType}&' +
+        'x=${scrollTop}&' +
+        'w=${screenHeight}&' +
+        'j=${decayTime}&' +
+        't=${pageViewId}&' +
+        'i=${title}&_',
+    },
+    'triggers': {
+      'trackInterval': {
+        'on': 'timer',
+        'timer-spec': {
+          'interval': 15
+        },
+        'request': 'interval'
+      }
+    },
+    'transport': {
+      'beacon': false,
```

Comment:
Currently, Chartbeat only supports image requests but we are looking into adding support for `sendBeacon` requests.